cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DFGPass.kt

```diff
@@ -138,6 +139,17 @@ class DFGPass(ctx: TranslationContext) : ComponentPass(ctx) {
         }
     }
 
+    /**
+     * Handle a [ThrowStatement]. Currently, we support two types of [ThrowStatement.exception],
+     * which are then forwarded to the appropriate handlers:
```

Comment:
Why is this mentioned here explicitly? For me it seems that this is completely opaque. We store whatever the expression handler gives us, so no need to restrict it.

Reply:
Removed